src/executor/alter_executor.cpp

```diff
@@ -0,0 +1,80 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// alter_executor.cpp
+//
+// Identification: src/executor/alter_executor.cpp
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "executor/alter_executor.h"
+
```

Comment:
[Trivial] Useless blank line here

Reply:
We are just following the Google C++ style here [https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes]

Reply:
ok, so it's an auto-format change, I see! Thanks